tests/foreman/ui/test_organization.py

```diff
@@ -35,12 +35,26 @@ def test_positive_end_to_end(session):
 
     :CaseLevel: Integration
 
-    :CaseImportance: High
+    :CaseImportance: Critical
```

Comment:
Merging everything will reduce the overall test execution time, but the single test execution time will increase, should we keep them in tier4 tests? (based on the time of execution)

Reply:
Hi @omkarkhatavkar  I'd say it's still not so lengthy to qualify for tier4, and I'd like to get these tests executed sooner, given the gating logic we'd like to implement. If this causes delays in tier2 we can move it later.

Reply:
agreed! thanks for the update.